README.md

```diff
@@ -15,6 +15,21 @@ sudo apt-get install build-essential m4 openjdk-11-jdk libgmp-dev libmpfr-dev pk
 pip install virtualenv poetry
 ```
 
+### MacOS (12.6 Monterey)
+
+Follow instructions for K's dependencies in MacOS from <https://github.com/kframework/k>. However, there are some remarks regarding:
+
+- llvm: make sure you install llvm@14, not llvm@15 or later, unless K supports it.
+- openssl: make sure you install openssl@3, or the version required by K. Your system may ship with openssl@1.1.
+- secp256k1: you may need to build it from
+  [source](https://github.com/bitcoin-core/secp256k1). For that you will
+  need to install GNU's autotools using homebrew.
+  * `brew install autoconf`
+  * `brew install automake`
+  While building secp256k1, make sure you run:
+  * `./configure  --enable-module-recovery`
+  in the appropriate step.
```

Comment:
We can add a make target to do this for the user in the same way [kevm does it](https://github.com/runtimeverification/evm-semantics/blob/cc9e2aac7e5e0a24a80c2f6c0a463c2b20a0e545/Makefile#L86-L91).

Reply:
We are moving towards different methods for installing K and its dependencies, such as `nix`. I think this is good as it stands.